src/app/app.module.ts

```diff
@@ -29,6 +28,7 @@ import { FooterComponent } from './footer/footer.component';
 import { HeaderComponent } from './header/header.component';
 import { PhoneComponent } from './phone/phone.component';
 import { LayoutModule } from '@angular/cdk/layout';
+import { DragToSelectModule } from '../../projects/ngx-drag-to-select/src/lib/drag-to-select.module';
```

Comment:
Do we want to import from `ngx-drag-to-select` here?
If so I think we'll have to modify the build process I think.